test/widget_test.dart

```diff
@@ -13,7 +14,7 @@ import 'package:demo/main.dart';
 void main() {
   testWidgets('Counter increments smoke test', (WidgetTester tester) async {
     // Build our app and trigger a frame.
-    await tester.pumpWidget(MyApp());
```

Comment:
probably this test will fail :)
you can remove it for now.

Reply:
commented out :)